app/helpers/projects_helper.rb

```diff
@@ -55,4 +55,10 @@ def healthy_count_class(count)
     else 'red'
     end
   end
+
+  def blocked(project)
+    blocks = project.deploy_blocks.unresolved.to_a
```

Comment:
Do we expect that a project might have more than one unresolved deploy block?

What do you think about limiting projects to at most one unresolved deploy block?

As a client initiating a block via the API, I feel like you should get a `422 Unprocessable Entity: A deploy block already exists for this project: <description>` error if one already exists.

```ruby
if (deploy_block = project.unresolved_block)
  tag.div(raw('&#9888;'), class: 'blocked', title: block.description)
end
```

cc/ @dleve123 @zephraph

Reply:
I dunno I think the initial modeling continues to make sense. If I create a block because X needs QA and you record another because of a critical bug discovered in staging, they're distinct and _both_ should be resolved before the project is cleared for release.

(I do agree that "whether a project is blocked at all" is an important higher level attribute of the project though.)

Reply:
Gotcha! Makes sense :+1: